src/js/inphinity.js

```diff
@@ -193,7 +193,10 @@ var inphinity = (function() {
             if (config.animationSpeed === "fast")
                 this.defaults.animationSpeed = 300;
         }
-        this.init();
+        
+        if(!!document.querySelector(config.navSelector)) {
```

Comment:
-1
This code should be sent to [inphinity.js](https://github.com/raphamorim/inphinity) repository.